test/tests/mesh_modifiers/boundingbox_nodeset/boundingbox_nodeset_inside_test.i

```diff
@@ -12,8 +12,8 @@
   [./middle_node]
     type = BoundingBoxNodeSet
     new_boundary = middle_node
-    top_right = '1 1 0'
-    bottom_left = '0.5 0.5 0'
+    top_right = '1.1 1.1 0'
+    bottom_left = '0.49 0.49 0'
```

Comment:
Another option rather than manually twiddling with the bounding box is to use the bounding box inflation routine. Uses could specify an inflation percentage. How do you plan to use this?